src/instructlab/eval/mt_bench_branch_generator.py

```diff
@@ -22,10 +23,17 @@
 def get_file_paths(directory):
     logger.debug(locals())
     file_paths = []
-    for root, _, files in os.walk(directory):
-        for file in files:
-            if file.split("/")[-1] == "qna.yaml":
-                file_paths.append(os.path.join(root, file))
+    root_path = Path(directory)
+    root_paths = [
+        entry
+        for entry in root_path.iterdir()
+        if entry.is_dir() and entry.name != "knowledge"
+    ]
+    for basedir in root_paths:
+        for root, _, files in os.walk(basedir):
+            for file in files:
+                if file.split("/")[-1] == "qna.yaml":
```

Comment:
(Observation) I think `files` are already stripped of parent path, so this check should probably just directly compare `file` to `"qna.yaml"` without `.split()`.

Reply:
That's a good catch. It would just be the file name sans directory.